spec/math/statistics/vector_spec.rb

```diff
@@ -92,6 +92,9 @@
       context "#mode" do
         it "returns the mode" do
           @dv.mode
+          expect(@dv.mode).to eq(666)
+          mode_test_example = Daru::Vector.new [1,2,3,2,4,4,4,4], dtype: dtype
+          expect(mode_test_example.mode).to eq(4)
         end
```

Comment:
Thank you very much for this fix. Just one concern:

Could you please rewrite this whole test as:
``` ruby
mode_test_example = Daru::Vector.new [1,2,3,2,4,4,4,4], dtype: dtype
expect(mode_test_example.mode).to eq(4)
```

The above 2 lines aren't necessary anymore.